README.md

```diff
@@ -60,25 +60,30 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Mnet Movie project  <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**Mnet Movie project** is a movie website with features of adding comment, tap on like to show interest and make reservations
+Here is a [presentation video](https://drive.google.com/file/d/1xt-lt4h_PEm0NxdSGNK2T7M4inuTJsKl/view?usp=sharing)
 
 ## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- Javascript
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+> Describe the tech stack and include only the relevant sections that apply to your project. -->
```

Comment:
- [ ] Please kindly remove all these templates from your README file in order to have a professional README file. 👍